dojo/tools/wazuh/parser.py

```diff
@@ -18,73 +25,59 @@ def get_label_for_scan_types(self, scan_type):
+                    item["condition"] != "Package unfixed"
+                    and item["severity"] != "Untriaged"
+                ):
+                    id = item.get("cve")
+                    package_name = item.get("name")
+                    package_version = item.get("version")
+                    description = item.get("condition")
+                    severity = item.get("severity").capitalize()
+                    agent_ip = item.get("agent_ip")
+                    links = item.get("external_references")
 
-        for item in vulnerability:
-            if (
-                item["condition"] != "Package unfixed"
-                and item["severity"] != "Untriaged"
-            ):
-                id = item.get("cve")
-                package_name = item.get("name")
-                package_version = item.get("version")
-                description = item.get("condition")
-                if item.get("severity") == "Untriaged":
-                    severity = "Info"
-                else:
-                    severity = item.get("severity")
-                if item.get("status") == "VALID":
-                    active = True
-                else:
-                    active = False
-                links = item.get("external_references")
-                title = (
-                    item.get("title") + " (version: " + package_version + ")"
-                )
-                severity = item.get("severity", "info").capitalize()
-                if links:
-                    references = ""
-                    for link in links:
-                        references += f"{link}\n"
-                else:
-                    references = None
+                    if links:
+                        references = "\n".join(links)
+                    else:
+                        references = None
 
-                if id and id.startswith("CVE"):
-                    vulnerability_id = id
-                else:
-                    vulnerability_id = None
+                    title = item.get("title") + " (version: " + package_version + ")"
+                    dupe_key = title
 
-                dupe_key = title
+                    if dupe_key in dupes:
+                        find = dupes[dupe_key]
+                    else:
+                        dupes[dupe_key] = True
 
-                if dupe_key in dupes:
-                    find = dupes[dupe_key]
-                else:
-                    dupes[dupe_key] = True
+                        find = Finding(
+                            title=title,
+                            test=test,
+                            description=description,
+                            severity=severity,
+                            mitigation="mitigation",
+                            references=references,
+                            static_finding=True,
+                            component_name=package_name,
+                            component_version=package_version,
```

Comment:
add cvssv3_score and publish_date here as well. Add dupe_key as unique_id_from_tool to prevent issues at deduplication.

`cvssv3_score = cvssv3_score,

publish_date = publish_date,

unique_id_from_tool = dupe_key,`